wordpress-mobile/gutenberg-mobile
# Introducing a Redux store

This is an exploratory PR for adding a Redux store to hold the block list.

All current list actions (focus, movement, deletion) is performed via Redux Actions: reducers recalculate the state when each event is received. The list then gets the state change via its (React) props.

Test plan:
* Run the Expo app and start using the toolbar buttons that appear after you focus a block (code block needs tapping on its header to be "focused" currently). All buttons except the "Settings" button should work
* Try editing the content of the core/code block. Any modifications get reverted but the console should print the edited text with the modification included. The block currently misses its 'onChange' callback and the modifications get reverted by default.
